Scrapers/Huone.py

```diff
@@ -0,0 +1,96 @@
+from bs4 import BeautifulSoup
+from requests import get
+import VenueClass
+import json
+from DataFormatting.FormatTime import FormatTime
+from DataFormatting.FormatPrice import FormatPrice
+from DataFormatting.FormatText import FormatTitle
+
+allVenues = []  # Array to store all Venues of type VenueClass
+
+def extractVenue(tag, venue):
+
+    # extract link into the venue
+    venue_link = venue.find_all('a')[0]['href']
+    response = get(venue_link)
+    indiv_venue = BeautifulSoup(response.text, 'html.parser')
+
+    # extract title
+    title_html = indiv_venue.find('div', class_='space-title').find('h1')
+    title = ''
+    if title_html:
+        title = FormatTitle(title_html.text.strip())
+
+    # location - fixed
+    location = 'HUONE Clarke Quay, 3D River Valley Road, #03-01, Clarke Quay, Singapore 179023'
+
+    # extract capacity
+    cap_html = indiv_venue.find('p', class_='persons-count')
+    capacity = ''
+    if cap_html:
+        capacity = cap_html.text.split(' ')[3].strip()
+
+    # extract description
+    desc_html = indiv_venue.find_all('div', class_='lead')[0]
+    desc = ''
+    if desc_html:
+        desc = desc_html.text.strip()
```

Comment:
I just noticed that the last sentence always doesnt have full stop. Maybe we can do a hard code **desc = desc_html.text.strip()+"."**